de.dlr.sc.virsat.external.lib.json-simple/about_files/AUTHORS.txt

```diff
@@ -1,3 +0,0 @@
-Yidong Fang

```

Comment:
I believe Apache license is quite strict regarding the formal roll-out. I think that we have to directly provide an authors.txt file. This file should not be removed.

Reply:
They deleted this file from the repository, now the author list is in the project page (to which we are linking in the about.html)